src/settings_import_export.py

```diff
@@ -39,19 +40,29 @@ def import_settings(path):
                     subprocess.run(['gsettings', 'set', \
                         'io.github.fsobolev.Cavalier', line.split(' ')[0], \
                         line.replace(line.split(' ')[0], '').strip()])
+        toast_msg = 'Settings sucessfully imported'
+
     except Exception as e:
         print('Can\'t import settings from file: ' + path)
         print(e)
+        toast_msg = 'Failed to import settings'
+
+    Adw.PreferencesWindow.add_toast(window, Adw.Toast.new(toast_msg))
```

Comment:
you can shorten this to `window.add_toast(Adw.Toast.new(toast_msg))`, same on line 68